puppet/modules/chassis/templates/local-config-extensions.php.erb

```diff
@@ -15,3 +15,7 @@ if ( file_exists( <%= path %> ) )
 	require_once <%= path %>;
 
 <%- end -%>
+
+<% if @subdomains != false -%>
```

Comment:
Should that be `<%- if`?

Reply:
Negative Sir. The `-` are for [space trimming](https://puppet.com/docs/puppet/5.3/lang_template_erb.html#space-trimming-1).